src/TypeScript.UnitTests/Notifications/UnsupportedNodeVersionNotificationServiceTests.cs

```diff
@@ -110,7 +110,7 @@ public void Show_ShowMoreInfoAction_OpensBrowser()
 
             firstAction.Action(null);
 
-            browserService.Verify(x=> x.Navigate("https://github.com/SonarSource/sonarlint-visualstudio/wiki"), Times.Once);
+            browserService.Verify(x=> x.Navigate("https://github.com/SonarSource/sonarlint-visualstudio/wiki/Supported-NodeJS-versions"), Times.Once);
```

Comment:
I think it should be "required" instead of "supported".

Reply:
Or perhaps "minimum". What do you think, @rita-sonarsource?

Reply:
Maybe something else entirely, like "nodejs-prerequisite"?

Reply:
Let's go with `NodeJs-prerequisite`.

Reply:
Or `NodeJS-prerequisite-for-JavaScript-and-TypeScript-analysis`, which makes the wiki page list look like this:

![image](https://user-images.githubusercontent.com/10757559/187393033-2e4a3a7d-1604-4fa1-ad16-291788855ced.png)

Just `NodeJS-prerequisite` on its own looked a bit odd, as it looked like it applied to all languages.

Reply:
LGTM 👍